src/components/Header/index.js

```diff
@@ -51,9 +51,8 @@ const Header = (props) => {
         <div className="HeaderLinksWrap">
           {match.path !== '/admin-login' && (
             <div className="HeaderLinks bold uppercase">
-              <Link to="#" className="HeaderLinkPricing">pricing</Link>
-              <Link to="#" className="HeaderLinkDocs">docs</Link>
-              <Link to="/login" className="HeaderLinkLogin TurnLight">login</Link>
+              <a href="http://staging-docs.cranecloud.io/" className="HeaderLinkDocs">Docs</a>
```

Comment:
this link is bound to change - it's actually dynamic... my suggestion is add it as a `const` in the `config` file and use it here

Reply:
in addition it may be used in a number of places

Reply:
This has been fixed @mrwagaba